src/Command/BuildDocsCommand.php

```diff
@@ -91,6 +91,12 @@ protected function configure()
                 InputOption::VALUE_NONE,
                 'Return a non-zero code if there are errors/warnings'
             )
+            ->addOption(
+                'print-errors',
+                null,
+                InputOption::VALUE_NONE,
+                'Print errors when to standard output'
```

Comment:
```suggestion
                'Print errors to standard output'
```

Maybe move this up under `save-errors`?

Or should we simply use the "verbosity" check - like maybe we print errors with `-v`? Or should we continue printing them... but NOT print them if you pass a `-q` for quiet? Sphinx DOES show errors while it's building (or, at least at the end... I can't remember exactly when it shows them)

Reply:
> Or should we continue printing them... but NOT print them if you pass a -q for quiet?

:+1: for this behavior. 

This is indeed consistent with Sphinx (which indeed shows errors directly while building).

Reply:
Hm.. But we build the `-vvv` argument. So I dont think it should be related to verbosity. 

> This is indeed consistent with Sphinx

That is true, but is that important?